src/test/scala/io/iohk/ethereum/vm/CallOpcodesSpec.scala

```diff
@@ -59,6 +59,10 @@ class CallOpcodesSpec extends WordSpec with Matchers with PropertyChecks {
       SELFDESTRUCT
     )
 
+    val selfDestructSelfCode = assemblyFromBytes (
```

Comment:
`Assembly` is meant to be a simple DSL, that enables to write EVM code that resembles it's actual assembly language and get the actual bytecode as a `ByteString`. Here we have a function that takes a `ByteString` and ultimately gives the same `ByteString` back, which is a bit pointless 😉 

`Assembly` can be improved to handle byte sequences. See the branch `fix/improvedAssemblyDSL`, and if you like the changes merge the latest commit into this branch.